docs/api/browser-view.md

```diff
@@ -1,5 +1,13 @@
 # BrowserView
 
+<!--
+```YAML history
+deprecated:
+  - pr-url: https://github.com/electron/electron/pull/35658
+    breaking-changes-header: deprecated-browserview
```

Comment:
Do we need to ensure that each of the `breaking-changes-header` values are unique? Otherwise I assume it'll only be possible to link to the top-most item.

Reply:
Both GitHub and Docusaurus will automatically de-duplicate heading slugs by appending a numeral to the end: https://gist.github.com/erickzhao/808369ac0f08d1d26adbc066eeb9c4ec

I think this actually raises a good point since we add more recent breaking changes to the top of the doc. In the instance that we actually change an API in a breaking way multiple times, then the header slugs are no longer stable.

e.g.

`API changed: session.setPermissionCheckHandler(handler)` in Electron 13 currently has the `#api-changed-sessionsetpermissioncheckhandlerhandler` slug.

If we make another modification/breaking change to the same API in Electron 40, the initial heading slug will change to `#api-changed-sessionsetpermissioncheckhandlerhandler-1`.